app/views/govuk_component/docs/search.yml

```diff
@@ -0,0 +1,19 @@
+name: "Search"
+description: "Search box"
+fixtures:
+  default: {}
+  for_use_on_a_dark_but_not_black_background:
+    on_dark_background: true
+  change_label_text:
+    label_text: "Search all the things"
+  stop_the_label_appearing_inline:
+    inline_label: false
+  apply_custom_styling_to_the_label:
+    inline_label: false
+    label_text: "<span class=\"fixture-title\">Search this website</span>"
```

Comment:
Re: fixtures, the component currently is only built for use on the homepage (although I have included options to make it suitable for use elsewhere), but yes, we should expand it to include further fixture options later.